svelte 4 support

When trying to update our sveltekit project (to use svelte4), we have this error:

> npm ERR! While resolving: MYPROJECT@1.2.1
> npm ERR! Found: svelte@4.1.2
> npm ERR! node_modules/svelte
> npm ERR!   dev svelte@"^4.0.0" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer svelte@"^3.54.0" from @beyonk/gdpr-cookie-consent-banner@10.0.0
> npm ERR! node_modules/@beyonk/gdpr-cookie-consent-banner
> npm ERR!   dev @beyonk/gdpr-cookie-consent-banner@"^10.0.0" from the root project

we used the "official" `npx svelte-migrate@latest svelte-4` command.

There is a small PR proposal to fix this: https://github.com/beyonk-group/gdpr-cookie-consent-banner/pull/51

--- a/package.json
+++ b/package.json
@@ -28,7 +28,7 @@
     "!dist/**/*.spec.*"
   ],
   "peerDependencies": {
-    "svelte": "^3.54.0"
+    "svelte": "^3.54.0 || ^4.0.0"
   },
   "dependencies": {
     "js-cookie": "^3.0.1"

